builders/manage/sudo/pause-transactions.md

```diff
@@ -0,0 +1,108 @@
+
+To find out the names of the modules available in your runtime, you need to read the `construct_runtime!()` section in the `lib.rs` file of your custom project. If your appchain is based on one of the official templates, refer to the [lib.rs](https://github.com/moondance-labs/tanssi/blob/master/container-chains/runtime-templates/frontier/src/lib.rs){target=\_blank} of the EVM template or the [lib.rs](https://github.com/moondance-labs/tanssi/blob/master/container-chains/runtime-templates/simple/src/lib.rs){target=\_blank} of the Substrate template.
+
+The following snippet is an example of how the `construct_runtime!()` section looks like. The module names are those located to the left of the colon.
+
+```rust
+construct_runtime!(
+    pub enum Runtime
+    {
+        ...
+        Migrations: pallet_migrations = 7,
+        MaintenanceMode: pallet_maintenance_mode = 8,
+        TxPause: pallet_tx_pause = 9,
+        Balances: pallet_balances = 10,
+        Multisig: pallet_multisig = 16,      
+        ...
+   }
+```
+
+To identify the transaction names included in a module, you need to refer to its source code. Modules built in [Substrate](/learn/framework/overview/#substrate-framework){target=\_blank} identify their transactions using a macro `#[pallet::call_index(INDEX)]`, where `INDEX` is a number. In the case of a [built-in module](/builders/build/customize/adding-built-in-module/){target=\_blank}, the code is located within the [FRAME folder](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame){target=\_blank} of the Polkadot-SDK repository. For example, if you want to know about the transaction names in the `Balances` module, refer to its [lib.rs](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/balances/src/lib.rs){target=\_blank} file and look for the function names below the `#[pallet::call_index(INDEX)]` macros. The following snippet is the transaction `transfer_allow_death` of the module `Balances`, which is the one used as an example in this guide:
+
+```rust
+#[pallet::call_index(0)]
+pub fn transfer_allow_death(
+    origin: OriginFor<T>,
+    dest: AccountIdLookupOf<T>,
+    #[pallet::compact] value: T::Balance,
+) -> DispatchResult {
+    // Code
+    Ok(())
+}
+```
+
+## Pausing Transactions {: #pausing-transactions }
+
+As you know, the Sudo account can perform privileged functions, such as pausing and unpausing transactions. To pause a transaction, navigate to the **Developer** tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Make sure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
```

Comment:
Add a link to the sudo management page with all the things sudo can do in "can perform privileged actions"

```suggestion
As you know, the Sudo account can perform privileged actions, such as appchain upgrades, minting new tokens, and, in this case, pausing and unpausing transactions. 

To pause a transaction, navigate to the **Developer** tab of Polkadot.js Apps for your Tanssi appchain and click on **Sudo**. If you do not see **Sudo** in this menu, you have not associated the Sudo account with Polkadot.js Apps. Make sure that your [Sudo account is injected by your wallet and connected to Polkadot.js Apps](/builders/manage/sudo/sudo/#configuring-polkadotjs-apps){target=\_blank}. Then, take the following steps:
```